File: C++/freeCodeCamp.org_C++ Tutorial for Beginners/src/024_Classes & Objects/ClassesAndObjects.cpp
#include <iostream>

/* Simple C++ program that uses classes and objects to represent realworld entities */

using namespace std;

class Book // A class is a custom datatype template or blueprint
{
    public:
        string title;
        string author;
        int pages;
};

int main()
{
    Book book1; // An object is an instance of a class
    book1.title = "Harry Potter"; // The objects public attributes can be directly modified
    book1.author = "JK Rowling";
    book1.pages = 500;

    Book book2;
    book2.title = "Lord of the Rings";
    book2.author = "Tolkien";
    book2.pages = 700;

    cout << "Title:\t" << book1.title << endl; // Prints the title of the first book object

    cout << "Title:\t" << book2.title << endl; // Prints the title of the second book object

    return 0;
}

File: Python/002_OOP Basics/024_Classes & Objects/ClassesAndObjects.py
# Python program that uses classes and objects to represent realworld entities
class Book(): # A class is a custom datatype template or blueprint
    title="" # Attribute
    author=""
    pages=0

book1 = Book() # An object is an instance of a class
book2 = Book()

book1.title = "Harry Potter" # The objects attribute values can be directly modified
book1.author = "JK Rowling"
book1.pages = 500

book2.title = "Lord of the Rings"
book2.author = "Tolkien"
book2.pages = 700

print("\nTitle:\t", book1.title,"\nAuthor:\t",book1.author,"\nPages:\t",book1.pages) # Prints the attribute values of the first book object
print("\nTitle:\t", book2.title,"\nAuthor:\t",book2.author,"\nPages:\t",book2.pages) # Prints the attribute values of the second book object